k8s/upgrades/0.8.1-0.8.2/snapshotdata_upgrade.sh

```diff
@@ -0,0 +1,38 @@
+run_snapshotdata_upgrades()
+{
+    if [ $# -eq 1 ]; then
+        pv=$1
+    else
+        echo "please pass persistentVolume name got pv: $pv"
+        exit 1
+    fi
+    # Get the list of volumesnapshotdata related to given PV
+    volumesnapshotdata_list=$(kubectl get volumesnapshotdata\
+                               -o jsonpath="{range .items[?(@.spec.persistentVolumeRef.name=='$pv')]}{@.metadata.name}:{end}")
+    rc=$?
+    if [ $rc -ne 0 ]; then
+        echo "failed to get snapshotdata name list"
+        exit 1
+    fi
+
+    if [ ! -z $volumesnapshotdata_list ]; then
+
+        pv_size=""
+        pv_size=$(kubectl get pv $pv -o jsonpath='{.spec.capacity.storage}')
+        rc=$?
+        if [ $rc -ne 0 ]; then
```

Comment:
check for empty or null pv_size also

Reply:
pv_size cann't be empty or null but it's good to have a check. I will add separate commit.